src/webgpu/util/constants.ts

```diff
@@ -339,7 +437,7 @@ export const kValue = {
       e: hexToF32(kBit.f32.positive.e),
       first_f64_not_castable: hexToF32(kBit.f32.positive.max) / 2 + 2 ** 127, // mid point of 2**128 and largest f32
```

Comment:
For a followup CL.  It's not entirely clear what this constant is trying to capture.

Is it the number that is just outside the finite range of f32 such that it's the smallest f64 that would convert to an f32 infinity?   I guess I'm not clear on the derivation.